build.py

```diff
@@ -181,6 +181,8 @@ def core_cmake_args(components, backends, install_dir):
         cmake_enable('gcs' in FLAGS.filesystem)))
     cargs.append('-DTRITON_ENABLE_S3:BOOL={}'.format(
         cmake_enable('s3' in FLAGS.filesystem)))
+    cargs.append('-DTRITON_ENABLE_AZURE_STORAGE:BOOL={}'.format(
+        cmake_enable('as' in FLAGS.filesystem)))
```

Comment:
Please replace 'as' with 'azure' to make it more intuitive.

Reply:
As @deadeyegoodwin mentioned [here](https://github.com/triton-inference-server/server/pull/2105#discussion_r504228531), 'azure' is too generic.

Reply:
I agree **azure** is too generic. I prefer **azure_storage** since **as** does not appear to be a standard acronym for Azure Storage.

Reply:
done.